### ISSUE TITLE
Switches no longer works for Chrome

### ISSUE THREAD

**Issue by darkogj** [watir/watir#613]
I've updated to the latest Watir and Chromedriver, and tried using the switches option:

http://watir.com/docs/browser-proxies/

But the browser doesn't respect the switches options.

Did something change for the way Chrome handles options? Or maybe it's Watir issue?

**Comment by darkogj** [comment]
I've confirmed this is a Watir issue. I have multiple versions on my PC. I've tried using 6.2.1 and Chrome switches work as intended.

**Comment by darkogj** [comment]
Me again. Found the problem, switches is depreciated. You might want to update the documentation:
```

options = Selenium::WebDriver::Chrome::Options.new
options.add_argument("--proxy-server=#{proxy_ip}:#{proxy_port}")
browser = Watir::Browser.new :chrome, options: options

```

**Comment by titusfortner** [comment]
What code did you use? Watir 6.6.2?

I changed things for a recent release to minimize the need for using selenium classes, but it should be backward compatible.

As for docs, I'm working on updates https://github.com/watir/watir.github.io/pull/95

Writing docs is not the fun part. :-)

**Comment by darkogj** [comment]
The latest. For 6.2.1 I got a selenium depreciation warning of "--switches" and to use the above code instead.

**Comment by titusfortner** [comment]
6.6.2 was released yesterday with a fix relating to capabilities, but probably not this specifically.

What code did you use to initialize the browser that gave an error?

I should figure out how to prevent users from getting the selenium deprecation notice.

**Comment by darkogj** [comment]
For 6.2.1 I got the warning related to 'switches'. For the latest version 6.6.3 this syntax no longer works:

`Watir::Browser.new :chrome, :switches => ["--proxy-server=#{proxy_ip}:#{proxy_port}"]`

**Comment by titusfortner** [comment]
fixed in master, will be included in 6.7 release soon.

**Comment by titusfortner** [comment]
Update that 6.7 has been released
